src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLQueryTypeImpl.java

```diff
@@ -776,12 +776,12 @@ public DataFetcher<DataFetcherResult<RoutingResponse>> plan() {
         (Collection<String> v) -> vehicleRental.setBannedNetworks(new HashSet<>(v))
       );
 
-      if (request.vehicleRental && !hasArgument(environment, "bikeSpeed")) {
-        //slower bike speed for bike sharing, based on empirical evidence from DC.
-        // TODO - There should be a separate speed preference for rented bike, setting this
-        //      - here will cause the different APIs to behave differently
-        preferences.withBike(b -> b.setSpeed(4.3));
-      }
+      // if (request.vehicleRental && !hasArgument(environment, "bikeSpeed")) {
```

Comment:
Is this intentionally removed?

Reply:
This hasn't worked since OTP migrated to separate modes for access/egress/transfer, as the value for `request.vehicleRental` is set only much later, in the `getStreetSearchRequest`, which was called by the individual routing phases. I added the todo, as I plan to implement form factor/propulsion specific speeds for rentals later on.